docs/source/literal_types.rst

```diff
@@ -23,8 +16,9 @@ precise type signature for this function using ``Literal[...]`` and overloads:
 
 .. code-block:: python
 
-    from typing import overload, Union
-    from typing_extensions import Literal
+    # Note: if you are using Python 3.7 or earlier, you will need to import
+    # Literal from the typing_extensions module.
```

Comment:
I would make it a proper `.. note::` (for example above at the start), we do this for `Protocol` for example.